mods/seasonal/moves.js

```diff
@@ -3209,10 +3211,10 @@ let BattleMovedex = {
 			// Generate new set
 			const generator = new RandomStaffBrosTeams('gen7randombattle', this.prng);
 			let set = generator.randomSet(claim);
+			// Suppress Ability now to prevent starting new abilities when transforming
+			source.addVolatile('gastroacid', source);
```

Comment:
This is undone when he switches out though, the question becomes will the ability be correct then too?

Reply:
*discord conversation*